blocks/init/src/Blocks/components/modal/modal.php

```diff
@@ -76,8 +71,7 @@ class="<?php echo \esc_attr($modalExitButtonClass); ?>"
 
 			<div class="<?php echo \esc_attr($modalContentClass); ?>">
 				<?php
-					// phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
-					echo $modalContent; // @phpstan-ignore-line
+					echo $modalContent; // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
```

Comment:
These PHPStan errors can be ignored, fixed them in this PR: https://github.com/infinum/eightshift-libs-stubs/pull/4